lms/templates/student_profile/learner_profile.underscore

```diff
@@ -13,6 +13,8 @@
             <i class="fa fa-exclamation-triangle message-error" aria-hidden=true></i>
             <span class="copy"><%- gettext("An error occurred. Please reload the page.") %></span>
```

Comment:
@Kelketek Ugh, sorry I missed this: "An error occurred. Try loading the page again."